src/runtime/testing/slot-test-observer.ts

```diff
@@ -55,7 +51,7 @@ export class SlotTestObserver extends AbstractSlotObserver {
     this.logging = Boolean(options.logging);
     this.debugMessages = [];
     // Clear all cached templates
-    HeadlessSlotDomConsumer.clearCache();
+    //HeadlessSlotDomConsumer.clearCache();
```

Comment:
Checking, is this intentional (could have been forgotten). Comment on why/whether/when we need to restore or rework?

Reply:
All `SlotConsumer` classes and subclasses are removed. This commented line just escaped my clean-up. I'll fix it.